src/dodona/feedback/CloseTest.java

```diff
@@ -1,13 +1,21 @@
 package dodona.feedback;
 
+import static dodona.junit.JUnitJSON.PROPERTY_OUTPUT_CUTOFF;
+
 public class CloseTest extends AcceptedClose {
 
+    private static final int output_cutoff = Integer.getInteger(PROPERTY_OUTPUT_CUTOFF, 1000);
+
     private String generated;
     private StatusPair status;
 
     public CloseTest(String generated, StatusPair status, boolean accepted) {
         super("close-test", accepted);
-        this.generated = generated;
+        if(generated.length() > output_cutoff) {
+            this.generated = generated.substring(0, output_cutoff) + "...";
```

Comment:
It probably makes sense to also add a message saying the user output was truncated, just to makes sure users don't think the output that Dodona shows is the same as the output that was generated.

Reply:
Would you suggest a separate "Message", or just a `+ "... (to much output)"` (i18ned)?

Reply:
I would do a separate message.